src/load.php

```diff
@@ -192,7 +192,7 @@ function exceptionHandler($e)
         echo $whoops->handleException($e);
     } else {
         include PATH_LIBRARY . DIRECTORY_SEPARATOR . 'Error' . DIRECTORY_SEPARATOR . 'ErrorPage.php';
```

Comment:
You've moved this file, this include path is no longer valid

Reply:
Nice catch - I'll update the file. 🙂